### PR TITLE
Required changes in CMake files

### PR DESCRIPTION
**Pull request by KulikovNikita**
Copy of [this commit](https://github.com/oneapi-src/oneDAL/pull/1731)

### CODE REVIEW

**Review by PetrovKP — APPROVED** [review]

**Review by michael-smirnov — APPROVED** [review]

File: examples/daal/cpp_sycl/CMakeLists.txt [hunk 1]
```diff
@@ -23,8 +23,13 @@ set(CMAKE_BUILD_TYPE Release)
 set(USE_DPCPP yes)
 set(USE_NEW_IFACES no)
 
+set(custom_options "-fsycl-device-code-split=per_kernel")
```
Comment (rlnx) [inline examples/daal/cpp_sycl/CMakeLists.txt:26]: This is out of style, CMake usually uses UPPER_CASE for variables, see how other variables defined

Comment (KulikovNikita) [reply examples/daal/cpp_sycl/CMakeLists.txt]: Fixed. Please see

**Review by rlnx — APPROVED** [review]

**Comment by KulikovNikita** [comment]
[Previous CI](http://intel-ci.intel.com/ebd3e2fc-3a97-f1a2-b2cf-8cdcd4b723a1)

**Pull request closed without merging**